### DISCUSSION TITLE
Shelly 2.5 ShutterOpenDuration set to 17.4 after backout

### DISCUSSION THREAD

**Discussion by score9** [arendst/Tasmota#16898] [Q&A] [answered]
Hello,
I have some Shelly 2.5 in my home in Shutter Mode to controll my window roller shutters.
Of course I have set the ShutterOpenDuration and ShutterCloseDuration to the specific values for each roller shutter.
But everytime a circiut breaker switches off (doesn't matter if due to a short or intended by me), nearly all of the shellys reset the ShutterOpenDuration and ShutterCloseDuration to 17.4 sec and I have to set the two values again.
I guess it has something to do with the wiring in this house but I cannot figuere out where it comes from.
Can anyone help me with why this happens or what I could do about it?

**Comment by stefanbode** [comment, marked as answer]
I' pretty sure you played with the option "savedata" and therefore avoid any storage of data after changes. "savedata 1" should bring you back into the game.

**Reply by score9** [reply]
you are right, nearly all shellys were set to savedata 0 - I can't remember that I have touched this setting.. but obviously I must have done it.
Also I found that one of my roller shutters takes 17.4 seconds to open. I must have set all roller shutters to this value accidentally.
Typical layer 8 problem I would say :)
Thanks for your help!

**Reply by sfromis** [reply]
`SaveData 0` is actually a good idea if you do not want the last state to be restored after power cycling. Obviously, that's not always true :wink: 
Just make sure to invoke a save after doing settings changes you want to be persistent.

**Reply by stefanbode** [reply]
I really advice not to use savedata 0. the driver capture current state of the shutter in variables. If you do not save on every reboot the position is a problem because set to the last position in savedata. I have never seen a problem with overuse of the flash.

**Reply by sfromis** [reply]
My prime reason for generally not saving state of devices like lights is simply that I do not want whatever state they were in at a power outage to be resumed when the power returns. Of course, shutters are a good reason for actually saving, as they have a physical state outside of what Tasmota can detect.
